src/app.module.ts

```diff
@@ -6,54 +6,58 @@ import { AppService } from './app.service';
 import { TweetsModule } from './tweets/tweets.module';
 import { UsersModule } from './users/users.module';
 import { AuthModule } from './auth/auth.module';
-
-// tslint:disable-next-line:no-var-requires
-const { ApolloServer } = require('apollo-server');
-// tslint:disable-next-line:no-var-requires
-const { importSchema } = require('graphql-import');
-// tslint:disable-next-line:no-var-requires
-const path = require('path');
+import { graphiqlExpress, graphqlExpress } from 'apollo-server-express';
+import { TypeOrmModule } from '@nestjs/typeorm';
```

Comment:
Can you please tell the reason of adding TypeOrmModule. Because we are using prisma orm

Reply:
Ahh, yeah, I haven't used Prisma in months. Forgot that was a thing. I'll go ahead and remove that

Accessing the database is something like `prisma.db.query.where({ id: args.where.id });`, right?

Reply:
yes I think so @juicycleff  any suggestions ?